net.blumia.pineapple.steam-recording-exporter.yml

```
@@ -0,0 +1,89 @@
+app-id: net.blumia.pineapple.steam-recording-exporter
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: PineappleSteamRecordingExporter
+rename-icon: net.blumia.pineapple.steam-recording-exporter
+
+finish-args:
+  # X11 and Wayland support
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  
+  # Audio/Video support
+  - --socket=pulseaudio
+  - --device=dri
```

```suggestion
  # X11 and Wayland support
  - --share=ipc
  - --socket=fallback-x11
  - --socket=wayland
  - --device=dri
  
  # Audio support
  - --socket=pulseaudio
```

the order doesn't matter, but given the unecessary comments and group at least make them thruthful